### ISSUE TITLE
Crash on startup using GroovyModLoader (GML)

### ISSUE THREAD

**Issue by Barerock** [ValkyrienSkies/Valkyrien-Skies-2#747]
### Mod Name

GroovyModLoader

### This issue occurs when only Valkyrien Skies, addons, and the mod I have specified are installed and no other mods

- [X] I have tested this issue and it occurs with only Valkyrien Skies, addons, and the mod I have specified

### Minecraft Version

1.20.x

### Mod Loader

Forge

### Issue description

Exit Code: 1 of course, so no crash log unfortunately. Almost instant crash when using MGL with ValkyrienSkies, Kotlin for Forge, Eureka, and GroovyModLoader (with no mods that use MGL, like EatingAnimations)

### Issue reproduction

Install above mods on Forge 47.2.20 client, launch.

### Logs

[latest.log](https://github.com/ValkyrienSkies/Valkyrien-Skies-2/files/14390689/latest.log)

No debug log available.

**Comment by Barerock** [comment]
https://github.com/GroovyMC/GroovyModLoader/issues/23

**Comment by Rubydesic** [comment]
Looks like this issue is caused by the unfathomably stupid way Forge has decided to set up "jar in jar". Nothing we can really do about this. I will leave the issue open until Forge fixes it though.

**Comment by mikedrugs** [comment]
getting this crash too

**Comment by AlienXtream** [comment]
still an issue nearly 2 years on. and BOY did it take some digging for me to find this... i assume that, given how early the crash occurs in the mod setup phase, its not possible to have VS2 flag GML as "incompatible"?

**Comment by blockninja124** [comment]
Correct. Same for VS telling people it requires kotlin for forge. Forge is dumb and crashes from these two issues before loading dependency data.
